src/rime/dict/dictionary.h

```diff
@@ -110,8 +110,8 @@ class DictionaryComponent : public Dictionary::Component {
                      vector<string> packs);
 
  private:
-  map<string, weak<Prism>> prism_map_;
-  map<string, weak<Table>> table_map_;
+  hash_map<string, weak<Prism>> prism_map_;
```

Comment:
This is a safe change. But I doubt if it contributes to any significant performance improvement.
If this is not on the critical code path, I'd rather not over-specify the type of map needed.

Reply:
This change do just get a little better performance. But I think any small improvement will contribute to the larger improvement, so I keep this.